cmd/tink-worker/cmd/root.go

```diff
@@ -105,16 +97,11 @@ func NewRootCommand(version string, logger log.Logger) *cobra.Command {
 	}
 
 	rootCmd.Flags().Duration("retry-interval", defaultRetryIntervalSeconds*time.Second, "Retry interval in seconds (RETRY_INTERVAL)")
-
 	rootCmd.Flags().Duration("timeout", defaultTimeoutMinutes*time.Minute, "Max duration to wait for worker to complete (TIMEOUT)")
```

Comment:
Since you added the change to support a non-timeout, maybe convey how to set that
```suggestion
	rootCmd.Flags().Duration("timeout", defaultTimeoutMinutes*time.Minute, "Max duration to wait for worker to complete. Set to '0s' for no timeout (TIMEOUT)")
```

Reply:
I was initially confused about this because I didn't remember messing with no-timeout stuff so went and checked. 0s for no timeout has been around for 2 years now. :+1: for making this better anyway though, will do.

Reply:
done